### PR TITLE
fix previous bad commit

### PR DESCRIPTION
**Pull request by lauterbur**
figure pdf is back, R code merge detritus removed, cleaned up R code in general

### CODE REVIEW

File: figs/species_fig.R [hunk 1]
```diff
@@ -225,14 +202,9 @@ tree_plot +
...
   scale_color_manual(name="count", na.value="white", values = c("grey","black")) +
-  # scale_x_discrete(position="top") +
   ggplot2::xlim(0, 8000) +
   ggplot2::ylim(-1.5,21.5)
 
 ggsave("./species_fig_temp6.png",height=9,width=9)
 
 # increase size of column labels, angle, remove "NA" in legend - do this outside R because ggtree doesn't want to
-=======
-dev.off()
```
Comment (petrelharp) [inline figs/species_fig.R:236]: This is necessary to close the device opened by the call to `pdf( )` way up there... but, I'm not sure how the figure's being made any more?

**Comment by petrelharp** [comment]
Does running this script produce the pdf file? (`species_fig.pdf`, I think?)

**Comment by petrelharp** [comment]
We should probably merge this in, @lauterbur?

**Comment by lauterbur** [comment]
Edits incorporated in https://github.com/popsim-consortium/adding-species-manuscript/pull/66

**Comment by lauterbur** [comment]
Ah, so since the packages I'm using to produce the base figure don't
*quite* do everything (adjusting the column label height without screwing
up the justification terribly, for example), I need to do some manual
manipulation after the figure is generated. I had left that as a png
(produced as "species_fig_temp6.png") and then converted to pdf in gimp
when I did the tweaks, but I can generate it as a pdf instead if that's
preferred.

**Pull request closed without merging**